grpc/proto/v1/search_get.proto

```diff
@@ -102,6 +102,21 @@ message ObjectPropertiesRequest {
   repeated ObjectPropertiesRequest object_properties = 3;
 }
 
+enum CombinationMethod {
+  COMBINATION_METHOD_UNSPECIFIED = 0;
+  COMBINATION_METHOD_TYPE_SUM = 1;
+  COMBINATION_METHOD_TYPE_MIN = 2;
+  COMBINATION_METHOD_TYPE_AVERAGE = 3;
+  COMBINATION_METHOD_TYPE_RELATIVE_SCORE = 4;
+  COMBINATION_METHOD_TYPE_MANUAL = 5;
+}
+
+message Targets {
+  repeated string targets = 1;
```

Comment:
can we keep the old name `target_vectors`?

Reply:
in the code it also brings `targets.Targets` maybe `targets.TargetVectors` would be more descriptive and inline with already used naming convention

Reply:
thought : what about `vectors` ? so it would be used `targets.Vectors`

Reply:
we usually use vectors for actual vectors (eg lists of numbers) - went with target vectors